{"name": "test_update_post[Valid - Update existing post]", "status": "failed", "statusDetails": {"message": "Failed: [ERROR] Invalid id from Excel: 'None'", "trace": "self = <tests.test_api_users.TestPostsAPI object at 0x000001E6CD7728B0>, data = {'action': 'update', 'body': 'updated title', 'expected': True, 'id': 1.0, ...}\n\n    @allure.description(\"Updates user details by user ID.\")\n    @pytest.mark.parametrize(*load_test_data(\"apis\", \"update\", parametrize=True))\n    def test_update_post(self, data):\n        expected = parse_bool(data.get(\"expected\"))\n        try:\n>           post_id = extract_post_id(data, fallback_id=self.__class__.created_ids[0])\nE           IndexError: list index out of range\n\ntests\\test_api_users.py:42: IndexError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <tests.test_api_users.TestPostsAPI object at 0x000001E6CD7728B0>, data = {'action': 'update', 'body': 'updated title', 'expected': True, 'id': 1.0, ...}\n\n    @allure.description(\"Updates user details by user ID.\")\n    @pytest.mark.parametrize(*load_test_data(\"apis\", \"update\", parametrize=True))\n    def test_update_post(self, data):\n        expected = parse_bool(data.get(\"expected\"))\n        try:\n            post_id = extract_post_id(data, fallback_id=self.__class__.created_ids[0])\n        except Exception:\n>           pytest.fail(f\"[ERROR] Invalid id from Excel: '{data.get('post_id')}'\")\nE           Failed: [ERROR] Invalid id from Excel: 'None'\n\ntests\\test_api_users.py:44: Failed"}, "description": "Updates user details by user ID.", "attachments": [{"name": "‚è∞ Test Start Time", "source": "0eb3ca50-9728-4662-8f89-5d534a69a0d0-attachment.txt", "type": "text/plain"}, {"name": "üèÅ Test End Time", "source": "56150341-1c80-48bb-9b9d-008eaac54c02-attachment.txt", "type": "text/plain"}, {"name": "‚è±Ô∏è Test Duration", "source": "7fca854c-bef7-4e0c-86a7-97687f5aa07c-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "data", "value": "{'action': 'update', 'id': 1.0, 'body': 'updated title', 'id': 'updated body', 'title': 1.0, 'expected': True, 'note': 'Valid - Update existing post'}"}], "start": 1753213311806, "stop": 1753213311808, "uuid": "fe4a1fed-844f-4b20-b528-07a5f03a9976", "historyId": "29bf52bb6897028a22060fc4a212a387", "testCaseId": "16130e9d55726e09524cf02a0fdf9edc", "fullName": "tests.test_api_users.TestPostsAPI#test_update_post", "labels": [{"name": "epic", "value": "posts API"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup')"}, {"name": "tag", "value": "api"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_api_users"}, {"name": "subSuite", "value": "TestPostsAPI"}, {"name": "host", "value": "Saber"}, {"name": "thread", "value": "31768-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_api_users"}]}